src/index.html

```diff
@@ -11,7 +11,185 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form>
+      <fieldset class="persInfo">
+
+        <legend>Personal information</legend>
+        Surname:
+        <input
+        type="text"
+        name="lastname"
+        autocomplete="off"
+        required
+        >
```

Comment:
please pay attention to attributes. they have to start from a new line each with a 2-space indentation. 
```suggestion
        <input
          type="text"
          name="lastname"
          autocomplete="off"
          required
        >
```

Reply:
check other places please